### DISCUSSION TITLE
VDH Edge license on Windows & Mac?!

### DISCUSSION THREAD

**Discussion by Zeihold** [aclap-dev/video-downloadhelper#507] [License & Purchase requests]
### What's your issue?

Hi guys,

I recently started using VDH again (have not used it in a while) and since my Chrome version I purchased a few years back through the Chrome Web store (currently) doesn`t activate, I decided to purchase the Edge version yesterday on my main W11 machine using Edge browser.

I received the email with the license number and it works fine on my windows machine. But I also have a Macbook Air M1 that I`m using Edge on as well and wanted to use VDH on that machine as well but the license doesn`t seem to activate.

I couldn`t find any info (at first glance) that states that each license is bound to a single device, but I also couldn`t find an info that states the opposite. I`m pretty sure the original license I purchased worked on all Chrome browsers that you were signed into with the same account.

So I guess I had this underlying expectation for the Edge version as well.

Can someone clarify if a license is per device or per account?

### The 10 first digits (only 10) of your license number or email address

e7a6ff00ab

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by Zeihold** [comment]
Funny enough, right after posting this here and trying for the x-th times in my Edge browser on my mac, the license number was accepted without an issue. So I guess the license is "per account" rather than per device. ;)

**Comment by paulrouget** [comment]
Licenses are not bound to a device.

It's one license by email address.

So your VDH Edge license would work on Edge on any device. We will lock your license if it's used on too many places though (you would get an email if that were to happen).
